docs/topics/coding-conventions.md

```diff
@@ -41,12 +41,54 @@ files in `org.example.kotlin.network.socket` should be in the `network/socket` s
 If a Kotlin file contains a single class or interface (potentially with related top-level declarations), its name should be the same
 as the name of the class, with the `.kt` extension appended. It applies to all types of classes and interfaces.
 If a file contains multiple classes, or only top-level declarations, choose a name describing what the file contains, and name the file accordingly.
-Use [upper camel case](https://en.wikipedia.org/wiki/Camel_case) with an uppercase first letter (also known as Pascal case),
+Use [an upper camel case](https://en.wikipedia.org/wiki/Camel_case) with an uppercase first letter (also known as Pascal case),
 for example, `ProcessDeclarations.kt`.
 
 The name of the file should describe what the code in the file does. Therefore, you should avoid using meaningless
 words such as `Util` in file names.
 
+In multiplatform projects, files with top-level declarations in platform-specific source sets should have a suffix
```

Comment:
I think Coding conventions for libraries is a separate section because it has general recommendations. Here we have specific tips for file naming, so it belongs to the File naming sections. I added a separate header for anchoring